core/unit_test/TestViewSubview.hpp

```diff
@@ -2110,15 +2110,30 @@ void test_unmanaged_subview_reset() {
 template <std::underlying_type_t<Kokkos::MemoryTraitsFlags> MTF>
 struct TestSubviewMemoryTraitsConstruction {
   void operator()() const noexcept {
-    using view_type          = Kokkos::View<double*, Kokkos::HostSpace>;
-    using size_type          = view_type::size_type;
     using memory_traits_type = Kokkos::MemoryTraits<MTF>;
+    using view_type =
+        Kokkos::View<double*, Kokkos::HostSpace, memory_traits_type>;
+    using size_type = typename view_type::size_type;
 
-    view_type v("v", 7);
+    Kokkos::View<double*, Kokkos::HostSpace> v_original("v", 7);
```

Comment:
Why did you introduce `v_original`

Reply:
I can't use `Unmanaged` with a managed `View`.

Reply:
Please add a comment.  This is a bit confusing.  All these views are "runtime unmanaged"...